Mathlib/Data/ZMod/Basic.lean

```diff
@@ -491,6 +491,14 @@ noncomputable def ringEquiv [Fintype R] (h : Fintype.card R = n) : ZMod n ≃+*
   RingEquiv.ofBijective _ (ZMod.castHom_bijective R h)
 #align zmod.ring_equiv ZMod.ringEquiv
 
+/-- The unique ring isomorphism between `ZMod p` and a ring `R` of cardinality a prime `p`. -/
```

Comment:
Would it be possible to add the uniqueness statement, so that if `e` is an equivalence between two such rings then `e=ringEquivOfPrime`?

Reply:
Rather than duplicating all the results about `ZMod.ringEquiv` (uniqueness, reflexivity, transitivity etc), I've added a comment in the definition of `ringEquivOfPrime` to explain how to turn it into `ZMod.ringEquiv` (the results for it are already there).